modules/commands/index.js

```diff
@@ -124,6 +127,14 @@ module.exports = function (client) {
 
     // If no command found, throw an error
     if (!item) {
+      // Check if they slightly misspelt a command and give a hint
+      const matches = stringSimilarity.findBestMatch(trigger, commandNames);
+      const bestMatch = matches?.bestMatch;
+      if (bestMatch.rating >= config.similaritySensitivity ?? 0.5) {
+        await message.channel.send(`Did you mean \`!${bestMatch.target}\`?\nType \`!help\` for a list of commands.`);
```

Comment:
What I was trying to get at with my previous comment here, is something like this:

```js
let response = `Sorry! I do not understand the command \`${trigger}\` `;
const matches = stringSimilarity.findBestMatch(trigger, commandNames);
const bestMatch = matches?.bestMatch;
if (bestMatch.rating >= config.similaritySensitivity ?? 0.5) {
  response += `Did you mean \`${bestMatch.target}\`?`;
}
response += '\nType `!help` for a list of commands';
await message.channel.send(response);
return;
```